packages/@lwc/features/src/rollup-plugin/index.js

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright (c) 2018, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: MIT
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/MIT
+ */
+const babel = require('@babel/core');
+const babelPlugin = require('../babel-plugin');
+
+module.exports = function rollupLwcFeatures(options = {}) {
```

Comment:
I would rather use https://github.com/rollup/rollup-plugin-babel and use the transformer in this package than creating here a rollup plugin.